UPtables/db/migrate/20150225204249_create_timetable_overfull_rooms.rb

```diff
@@ -0,0 +1,13 @@
+class CreateTimetableOverfullRooms < ActiveRecord::Migration
+  def change
+    create_table :timetable_overfull_rooms do |t|
+      t.references :timetable, index: true
+      t.references :entry, index: true
+      t.integer :severity
+
+      t.timestamps null: false
+    end
+    add_foreign_key :timetable_overfull_rooms, :timetables
+    add_foreign_key :timetable_overfull_rooms, :timetable_entries
```

Comment:
@felixKubicek this seems to be rails default so I'm going to stick to it. This time scaffolded with ```rails g model a_key:references```